### DISCUSSION TITLE
Clearing stored parameters

### DISCUSSION THREAD

**Discussion by QuirkyCort** [jscad/OpenJSCAD.org#1325] [General]
I noticed that when using parameters, JSCAD stores the values in localstorage using the parameter name as key, and reloads it (...overriding the default in getParameterDefinitions) on the next load. This occurs even when a different file is loaded via the URL, so changes to the parameter for one file will affect the next file.

Example:

1) Load this URL: https://www.openjscad.xyz/?uri=https://raw.githubusercontent.com/QuirkyCort/misc/main/jscad_models/vl53l0x.js

2) Change the height parameter from the initial value of 15 to 50.

3) Load this other URL: https://www.openjscad.xyz/?uri=https://raw.githubusercontent.com/QuirkyCort/misc/main/jscad_models/gy33.js

4) The default height value is now 50 for the second file, instead of the default of 18.

Is there a way to force JSCAD to always load the defaults specified in getParameterDefinitions?

**Comment by hrgdavor** [comment]
adding this to your script after imports could work:

```js
localStorage.setItem('jscad:jscad1-design','{}')
```

**Comment by QuirkyCort** [comment]
I got a "localStorage is not defined". I'm guessing the script is running in a webworker where localStorage isn't available?

**Comment by hrgdavor** [comment]
oh, dammit. Yes it is running in the worker. Guess that only option is to manually use Reset button.
